src/header.js

```diff
@@ -5,53 +5,35 @@ class Header extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      searchActive: "",
+      searchActive: false,
```

Comment:
rather than storing the classname in state, I think it's better to store it as a boolean, and apply the classname during render (see my change to the render function to see the rest of my approach)